Foundation/NSPathUtilities.swift

```diff
@@ -30,7 +30,11 @@ public func NSTemporaryDirectory() -> String {
             return tmpdir
         }
     }
+    #if os(Android)
```

Comment:
Just trying to follow the existing guards in the file.. Should I change those too?